hw03_woodd_house.py

```diff
@@ -0,0 +1,99 @@
+#################################################################################
+# Username:woodd
+
+# Google Doc Link: https://docs.google.com/document/d/1hR3vQZg9h3VkzSt2vV9erAGYUh89hPb0rmL-ib4MJRc/edit?usp=sharing
+################################################################################
+
+import turtle
+wn = turtle.Screen()
+wn.bgcolor("lightblue")
+
+wn.colormode(255)
```

Comment:
There should be no code outside of functions anymore. Move these three lines to the beginning of your `main` function.